5.2curlcommands.md

```diff
@@ -671,6 +671,55 @@ curl -X POST -H 'Content-type: application/json' http://localhost:8080/api/admin
 }'
 ```
 
+### H5Web Previewer for HDF5 and NetCDF files:
+The source can be found at https://github.com/JR-1991/hdf5-previewer

```

Comment:
```suggestion
```
We should delete this, right? Now the source is in this PR, right?

Reply:
True, just committed the suggestion to the PR.